Dockerfile

```diff
@@ -32,4 +32,4 @@ ENV INSTITUTION=unknown
 HEALTHCHECK --interval=20s --timeout=1s \
   CMD curl -f http://localhost:80/ping
 
-CMD node ./dist/source/ccc-server/index.js
+CMD npm run start:prod
```

Comment:
I saw an article recently which said you should not use npm-run as your ENTRYPOINT or CMD, because it doesn't send signals to the underlying command.